src/ontology/components/cob-to-external.tsv

```diff
@@ -92,6 +92,7 @@ COB:0000076	objective specification	owl:equivalentClass	IAO:0000005	objective sp
 COB:0000079	plan specification	owl:equivalentClass	IAO:0000104	plan specification	.
 COB:0000080	complex of molecular entities	rdfs:subClassOf	owl:Thing	owl:Thing	
 COB:0000088	drug product	owl:equivalentClass	DRON:0000005	drug product	we currently use a different text definition but we believe they do not contradict
+COB:0000088	drug product	rdfs:seeAlso	CHEBI:23888	drug	These can't be equivalent because COB drug product is a material entity, and ChEBI's drug is a role. See issue #140 for more information.
```

Comment:
I like this. closeMatch is also not wrong, but I think seeAlso is good as it's even weaker. We should systematize this a bit more